### PR TITLE
removed references to s3

### PR DESCRIPTION
**Pull request by sei-aschlackman**

### CODE REVIEW

**Review by sei-tspencer — APPROVED** [review]

File: .github/workflows/client.yml [hunk 1]
```diff
@@ -20,10 +20,10 @@ jobs:
       - name: Setup Package Name
         id: package_name
         run: |
-          echo ::set-output name=name::S3.Player.Api.Client
+          echo ::set-output name=name::Player.Api.Client
           echo ::set-output name=packagename::Player.Api.Client
```
Comment (sei-ebram) [inline .github/workflows/client.yml:24]: can remove this now

**Comment by sei-ebram** [comment]
Also, the internal deployment repo will need updated, pretty sure the `docker command` references s3

**Review by sei-chershberger — APPROVED** [review]

**Review by sei-noconnor — APPROVED** [review]

File: .github/workflows/client.yml (hunk 1 above)
Comment (sei-aschlackman) [reply .github/workflows/client.yml]: Updated, if you wanna double-check

**Review by sei-ebram — APPROVED** [review]

File: .github/workflows/client.yml (hunk 1 above)
Comment (sei-ebram) [reply .github/workflows/client.yml]: lgtm

**Pull request merged**